### ISSUE TITLE
Encountered a lil bug on the Electron Fiddle Page

### ISSUE THREAD

**Issue by Jassi10000** [electron/fiddle#1005]
![image](https://user-images.githubusercontent.com/64856348/157676171-067a08b6-7822-4896-97f7-d0a13af47a0c.png)

On clicking the Github Icon , the page should be directed to  
https://github.com/electron/fiddle

But instead it redirects to
https://github.com/electron
which I think needs to be changed , would love to hear views of the maintainers on this , if they allow i will even contribute to this one

**Comment by erickzhao** [comment]
I don't think this is valid since it's the same navbar as everywhere on the site. IMO it'd be more confusing if this icon _did_ redirect to the Fiddle repo.
